spec/tree_spec.rb

```diff
@@ -3,12 +3,41 @@
 
 describe Tree do
   it 'should be a Class' do
-    expect(described_class.is_a? Class).to eq true
+    expect(described_class.is_a?(Class)).to eq true
+  end
+  it 'should be dead at the age of 11' do
+    10.times { subject.age! }
+    expect(subject).to_not be_dead
+    subject.age!
+    expect(subject).to be_dead
+  end
```

Comment:
Ideally, this would be two separate tests - one for dying at 11, another for not dying before then.